agent/util-scripts/unittests

```diff
@@ -31,7 +31,7 @@ mv ${_testopt}/config/pbench-agent.cfg ${_testopt}/config/pbench-agent.cfg.orig
 let res=res+${?}
 cp -a ${_tdir}/../templates/* ${_testopt}/templates/
 let res=res+${?}
-cp -a ${_tdir}/{require-rpm,validate-*,pbench-*} ${_testopt}/util-scripts/
+cp -a ${_tdir}/{pbench-*,require-rpm,validate-*} ${_testopt}/util-scripts/
```

Comment:
Did you change anything here other than the order (which is not significant...right?)?

Reply:
So the reason why this is changed is because of the iterative process used to develop these commits that occurred in parallel with PR #2212.

Initially I started out by mocking `require-rpm` for most tests, and then testing `require-rpm` directly, but then realized that I could just mock out `rpm` and make all the tests use `require-rpm`.  So before I rebased this PR (or really PR #2210) on PR #2212, when I removed `require-rpm` we just had `pbench-*` left over, so I drop the `{}` construct all together.

When I merged with PR #2212, I had to restore the `{}` construct, and then when I realized I only have to mock `rpm`, I added it back in sorted order.